crates/starknet-devnet-server/src/api/json_rpc/mod.rs

```diff
@@ -271,8 +275,71 @@ impl JsonRpcHandler {
             }
         }
 
+        // TODO is there a case when is_err but we need to dump?
```

Comment:
no such case

Reply:
Removed comment in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/572/commits/a43ff09cb4eb53fab8df747241fa5935e097bd30